src/scss/components/_breadcrumbs.scss

```diff
@@ -36,8 +39,10 @@
     .c-breadcrumb-item {
         float: left;
         color: $breadcrumb-text-color;
+        font-weight: $breadcrumb-text-weight;
 
-        svg {
+        svg,
+        svg path {
```

Comment:
Hey @dandel10n, sorry to drag this PR up again! Just wondering what the reasoning was behind adding the additional selector here? It's causing issues in menu as the specificity is quite high and the only way we can really override is by using `!important`. 

Should the colour actually be `$color-content-default`? Looking at the component designs in Figma it's actually set to `$color-content-light`.